bin/check-sqs-messages.rb

```diff
@@ -131,11 +137,14 @@ def run
       warn = false
       crit = false
       queues = []
+      exclusions = config[:exclude_queue].split(',')
```

Comment:
we also could have done this as a `proc` when the user inputs it, see https://github.com/sensu-plugins/sensu-plugins-aws/blob/16.2.0/bin/check-vpc-nameservers.rb#L41 for an example, I will update the code prior to release to keep the code more consistent.